File: src/Egroo.React/src/theme/theme.ts
import { createTheme } from '@mui/material/styles';

// Create a theme matching the current MudBlazor design
const theme = createTheme({
  palette: {
    mode: 'light',
    primary: {
      main: '#1976d2', // MudBlazor default primary color
      light: '#42a5f5',
      dark: '#1565c0',
    },
    secondary: {
      main: '#dc004e', // MudBlazor default secondary color
      light: '#ff5983',
      dark: '#9a0036',
    },
    background: {
      default: '#fafafa',
      paper: '#ffffff',
    },
  },
  typography: {
    fontFamily: '"Roboto", "Helvetica", "Arial", sans-serif',
    h1: {
      fontSize: '2.125rem',
      fontWeight: 400,
      lineHeight: 1.235,
    },
    h2: {
      fontSize: '1.5rem',
      fontWeight: 400,
      lineHeight: 1.334,
    },
    h3: {
      fontSize: '1.25rem',
      fontWeight: 400,
      lineHeight: 1.6,
    },
    h4: {
      fontSize: '1.125rem',
      fontWeight: 400,
      lineHeight: 1.5,
    },
    h5: {
      fontSize: '1rem',
      fontWeight: 400,
      lineHeight: 1.5,
    },
    h6: {
      fontSize: '0.875rem',
      fontWeight: 500,
      lineHeight: 1.6,
    },
  },
  components: {
    MuiAppBar: {
      styleOverrides: {
        root: {
          boxShadow: 'none',
          borderBottom: '1px solid rgba(0, 0, 0, 0.12)',
        },
      },
    },
    MuiCard: {
      styleOverrides: {
        root: {
          boxShadow: '0px 1px 3px rgba(0, 0, 0, 0.12), 0px 1px 2px rgba(0, 0, 0, 0.24)',
        },
      },
    },
  },
});

export default theme;

File: src/Egroo.UI/Constants/Theme.cs
using MudBlazor;

namespace Egroo.UI.Constants
{
    public static class Theme
    {
        public static readonly MudTheme DefaultTheme = new MudTheme()
        {
            PaletteDark = new PaletteDark()
            {
                Primary = "#F25922",
                PrimaryContrastText = "#FFFFFF",
                Secondary = "#F2B591",
                SecondaryContrastText = "#1a1a1a",
                Black = "#27272f",
                Background = "#32333d",
                BackgroundGray = "#27272f",
                Surface = "#1e1e1e",
                DrawerBackground = "#1e1e1e",
                DrawerText = "rgba(255,255,255, 0.50)",
                DrawerIcon = "rgba(255,255,255, 0.50)",
                AppbarBackground = "#1e1e1e",
                AppbarText = "rgba(255,255,255, 0.70)",
                TextPrimary = "rgba(255,255,255, 0.90)",
                TextSecondary = "rgba(255,255,255, 0.70)",
                ActionDefault = "#adadb1",
                ActionDisabled = "rgba(255,255,255, 0.26)",
                ActionDisabledBackground = "rgba(255,255,255, 0.12)",
                Divider = "rgba(255,255,255, 0.12)",
                DividerLight = "rgba(255,255,255, 0.06)",
                TableLines = "rgba(255,255,255, 0.12)",
                LinesDefault = "rgba(255,255,255, 0.12)",
                LinesInputs = "rgba(255,255,255, 0.3)",
                TextDisabled = "rgba(255,255,255, 0.2)",
            },
            Typography = new Typography()
            {
                Default = new DefaultTypography()
                {
                    FontFamily = ["Inter", "Roboto", "Helvetica", "Arial", "sans-serif"],
                    FontSize = "0.875rem",
                    FontWeight = "400",
                    LineHeight = "1.5",
                    LetterSpacing = "0.00938em",
                },
                H5 = new H5Typography() { FontWeight = "600" },
                H6 = new H6Typography() { FontWeight = "600" },
                Button = new ButtonTypography()
                {
                    FontWeight = "600",
                    LetterSpacing = "0.02em",
                    TextTransform = "none",
                },
            },
        };
    }
}
